ledger/src/blockstore_metrics.rs

```diff
@@ -511,6 +512,13 @@ pub(crate) fn report_rocksdb_write_perf(metric_header: &'static str) {
     });
 }
 
+#[derive(Debug, Default, Clone)]
+/// A struct that holds the current status of RocksDB perf sampling.
+pub struct BlockstoreRocksDbPerfSamplingStatus {
```

Comment:
Such long names actually reduce readability of the code.
Within this module and blockstore module it is clear that this is blockstore specific.
outside this module, this is already qualified by the name of the module:
```
blockstore_metrics::BlockstoreRocksDbPerfSamplingStatus
```
So there is no need to repeat `Blockstore` twice.
Also there is no use for mentioning `RocksDb` in the name. Does this struct care if the backend is rocks-db or something else?